source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -112,6 +112,9 @@ Removed features and their replacements
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
 | ``disable_plaintext_auth``                                 | This has been replaced with :dovecot_core:ref:`auth_allow_cleartext` setting.            |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| :dovecot_core:ref:`ssl=required <ssl>`                     | Connections from :dovecot_core:ref:`login_trusted_connections` are now also required to  |
```

Comment:
There is no login_trusted_connections label to refer to. Maybe this meant to use 'dovecot_core:ref:`login_trusted_networks`'

Reply:
whoops, indeed.